lib/engine/game/g_1889/game.rb

```diff
@@ -255,7 +255,7 @@ class Game < Game::Base
             abilities: [
               {
                 type: 'tile_lay',
-                when: %w[stock_round owning_player_or_turn or_between_turns],
+                when: %w[stock_round owning_player_track or_between_turns],
                 hexes: %w[B11 G10 I12 J9],
```

Comment:
as discussed in slack, changing to owning_player_track. 
If a player wants to use this ability after their track step they can use it during "or_between_turns"